src/task/__tests__/TaskRunnerIntTest.test.ts

```diff
@@ -0,0 +1,82 @@
+import { expect, describe, test, jest } from "@jest/globals";
+import {
+  OrkesApiConfig,
+  orkesConductorClient,
+  WorkflowExecutor,
+  TaskRunner,
+  simpleTask,
+} from "../../../";
+
+const config: Partial<OrkesApiConfig> = {
+  keyId: `${process.env.KEY_ID}`,
+  keySecret: `${process.env.KEY_SECRET}`,
+  serverUrl: "https://pg-staging.orkesconductor.com/api",
```

Comment:
I would take this also from env as we should avoid exposing our internal orkes endpoints.